### ISSUE TITLE
Cluster 'threat actor' - proposed new fields

### ISSUE THREAD

**Issue by certeu** [MISP/misp-galaxy#16]
For the threat actor cluster, I suggest the following additional fields that we use in our CTI db.
In our day to day activities, they are quite useful for filtering, statistics/trends, etc.

Field = Motive
Values = Cybercrime, Cyberwar, Espionage, Hacktivism, Hacktivism-Nationalists, Jihadism, Other, Unknown

Field = Type
Values = Indendent group, State or State-sponsored group, Individual, Other, Unknown

Field = First Seen
Value = Date

Field = Last Seen
Value = Date

**Comment by iglocska** [comment]
The galaxy datastructure already allows adding these, just add the key values to a threat actor's meta list in the json. We are looking forward to any pull requests containing those data points.

**Comment by certeu** [comment]
Fine, thanks

**Comment by adulau** [comment]
@certeu I did some manual fixes from your patch-1 available in your fork.  The updates are now merged into misp-galaxy:

https://github.com/MISP/misp-galaxy/commit/649c043ad27b1304f46810fba4e11742163cf8b0

For a first contribution, it was quite good. I would really advise to use ['jq'](https://stedolan.github.io/jq/) to test your JSON file and also format it in a pretty-print format which is easier for diffing from previous versions.

We are anxiously waiting for your next contribution. Thank you!
